.github/workflows/SlackNotification.yml

```
@@ -10,6 +10,8 @@ jobs:
   permissions:
    id-token: write  # to authenticate via OIDC
   uses: SonarSource/gh-action_build-notify/.github/workflows/main.yaml@v1
-  if: github.event.check_run.name == 'Sonar.Net'
+  if: >-
```

See https://github.com/SonarSource/sonar-dotnet/blob/9c5b2d14549c5630aa09b10666f261ac92f89f7e/.github/workflows/CreatePullRequest.yml#L14
And that's also what GH uses in their docs and examples
```suggestion
  if: |
```